provision/kubernetes/cluster.go

```diff
@@ -498,6 +500,17 @@ func (c *ClusterClient) SinglePool() (bool, error) {
 	return strconv.ParseBool(singlePool)
 }
 
+func (c *ClusterClient) AvoidMultipleServices() (bool, error) {
```

Comment:
What about `createAdditionalServiceForVersion`?
```suggestion
func (c *ClusterClient) createAdditionalServiceForVersion() (bool, error) {
```